src/actions/auth/actionTypes.js

```diff
@@ -0,0 +1,7 @@
+const RegistrationConstants = {
+  REGISTER_REQUEST: 'USERS_REGISTER_REQUEST',
+  REGISTER_SUCCESS: 'USERS_REGISTER_SUCCESS',
+  REGISTER_FAILURE: 'USERS_REGISTER_FAILURE',
+};
+
+export default RegistrationConstants;
```

Comment:
Exporting these as defaults is not a good idea, what if someone else adds LoginConstants

Reply:
Thanks this has been resolved.